src/coderdojochi/templates/about.html

```diff
@@ -10,6 +10,7 @@
 <div class="container">
     <h1>{% trans "About CoderDojoChi" %}</h1>
     <div class="row">
+        Students Served: {{ students_served_count }}
         <p>CoderDojoChi is a volunteer not for profit that teaches web, game, and app development to youth ages 7 to 17.</p>
         <p>Our mission is to create a fun, collaborative environment to explore <abbr title="STEM is an acronym referring to the academic disciplines of science, technology, engineering, and mathematics.">STEM</abbr>. Through challenging play, youth can build memorable experiences with mentors. All classes are 100% free to attend thanks to your generous <a title="Donate now" href="{% url 'donate' %}">donations</a>.</p>
         <p>Ali Karbassi founded CoderDojoChi in 2013. It has grown to more than {{ mentor_count }} active mentors teaching over {{ students_served }} students. We host twice monthly workshops with dozens of youth, and reach hundreds of families across the Greater Chicago area.</p>
```

Comment:
`{{ mentor_count }}` and `{{ students_served }}` need to be updated to be `{{ view.mentor_count }}` and `{{ view.students_served }}`